catroid/src/main/java/org/catrobat/catroid/common/Constants.java

```diff
@@ -33,7 +33,7 @@
 	// Reflection in testcases needed
 	// http://stackoverflow.com/questions/1615163/modifying-final-fields-in-java?answertab=votes#tab-top
 
-	public static final float CURRENT_CATROBAT_LANGUAGE_VERSION = Float.valueOf(0.993f);
+	public static final float CURRENT_CATROBAT_LANGUAGE_VERSION = Float.valueOf(0.994f);
```

Comment:
Side note: even if no changes are required, it should still raise the old project's version number, otherwise there would be problems for future language version conversions as every conversion would be stuck on 0.993